Documentation/network/clustermesh/clustermesh.rst

```diff
@@ -48,12 +48,12 @@ Additional Requirements for Native-routed Datapath Modes
 * Cilium in each cluster must be configured with a native routing CIDR that
   covers all the PodCIDR ranges across all connected clusters. Cluster CIDRs are
   typically allocated from the ``10.0.0.0/8`` private address space. When this
-  is a case a native routing CIDR such as ``10.0.0.0/9`` should cover all
+  is a case a native routing CIDR such as ``10.0.0.0/8`` should cover all
```

Comment:
```suggestion
  is the case a native routing CIDR such as ``10.0.0.0/8`` should cover all
```

Reply:
Don't be afraid to request changes for suggestions like this. Otherwise the MLH marks the PR as ready-to-merge like below and the PR may go in without addressing this :-)